client/verta/verta/_internal_utils/_utils.py

```diff
@@ -270,6 +270,16 @@ def _get_visible_orgs(self):
         org_names = filter(None, org_names)
```

Comment:
Just to double check: In permissions V2, `workspace.org_name` is still the accurate workspace name?

Reply:
yes